backend/src/App.ts

```diff
@@ -2,13 +2,14 @@ import dotenv from "dotenv";
 import express, { Application } from "express";
 import cors from "cors";
 import bodyParser from "body-parser";
-import routes from "./routes/index";
+import routes from "./routes";
 import { MongoClient } from "mongodb";
 import DbUpdateService from "./services/DbUpdateService";
 import CMSAuthService from "./services/CMSAuthService";
 import http from "http";
 import chalk from "chalk";
 import * as log from "loglevel";
+import PUBLICTutorialService from "./services/PUBLICTutorialService";
```

Comment:
why is it `PUBLIC` instead of `Public`. Make use of proper pascal case.

Reply:
I did this because we have CMSTutorial and PUBLICTutorial, it just helped me clarify but will change if you still want me to.

Reply:
CMS is acro where as public is not. so, may change it.

Reply:
I'll change it now broski

Reply:
The file names we can't change yet, I think we can rename them in master. So that'll have to be done after the merge lol. The actual variables, imports, etc, all done.

Reply:
great you can do that.